src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/LSPDocumentChangeKind.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.VisualStudio.LanguageServerClient.Razor
+{
+    public enum LSPDocumentChangeKind
+    {
+        Added,
+        Removed,
+        VirtualDocumentChanged,
```

Comment:
Do we also need one for "regular" changed events?

Reply:
We don't have a requirement for regular changed events yet so I didn't feel it was worth it quite yet to listen for those.